### PR TITLE
added six to setup.py

### PR DESCRIPTION
**Pull request by ronthalanki-wework**
Added to the install_requires section in setup.py

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -41,6 +41,7 @@
     packages=find_packages(),
     install_requires=[
         "requests"
+        "six"
```
Comment (wslulciuc) [inline setup.py:44]: @ronthalanki-wework : I think install requirements are comma (`,`) separated. Did you have any issues installing dependencies locally?

Comment (ronthalanki-wework) [reply setup.py:44]: yep, i needed to add a comma

Comment (wslulciuc) [reply setup.py:44]: Great!

**Review by wslulciuc — APPROVED** [review]
LGTM 👍

**Pull request merged**